internal/rbd/controllerserver.go

```diff
@@ -424,6 +466,19 @@ func (cs *ControllerServer) CreateVolume(
 		return nil, err
 	}
 
+	// Set rbd qos for image
+	if len(rbdVol.QosParameters) != 0 && rbdVol.Mounter == rbdNbdMounter {
```

Comment:
It probably makes sense to store the QosParameters even when the mounter isn't NBD. With the upcoming ControllerModifyVolume CSI call, users may possibly switch between different mounters in the future.

There is also interest in configuring IOPS for block devices (through `/sys`) when the kernel mounter is used. Basing the IOPS configuration on what NBD uses would be most practical.

Reply:
Get, as long as SC provides QoS parameters, we should set them for it. But, we need to clarify that currently only NBD is supported.